pkg/skaffold/errors/errors.go

```diff
@@ -106,3 +99,41 @@ func concatSuggestions(suggestions []*proto.Suggestion) string {
 	s.WriteString(".")
 	return s.String()
 }
+
+var allErrors = map[Phase][]problem{
```

Comment:
```suggestion
var defaultErrors = map[Phase][]problem{
```
?

Reply:
didn't realize `knownBuildProblems` and `knownInitProblems` was also being appended.

Reply:
No Op.